### PR TITLE
Add work and about page

### PR DESCRIPTION
**Pull request by Victoria129**
I have added the works and about pages using CSS and HTML.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,18 +26,149 @@ <h3 class="poppins"><a href="#">My Logo</a></h3>
...
+    <p class="poppins-regular">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="poppins-regular" id="languages">
+      <li class="languages">html</li>
+      <li class="languages">css</li>
+      <li class="languages">javascript</li>
+    </ul>
+    <button class="poppins-regular">See Project</button>
+  </section>
+  <br/>
+  <br/>
```
Comment (ridwanediallo) [inline index.html]: - [ ] Hey @Victoria129, you did a great job on this project.👍️ However, I would recommend you to wrap all your `projects sections` into one `section ` that you may call for instance `work-section` and use `display: grid`  on it to align the sections of your different projects. 
Like this 👇️
Screenshot from 2022-09-24 00-21-19

And in your css file you can do this 👇️
```
.work-section {
  display: grid;
}
```

File: style.css [hunk 2]
```diff
@@ -83,4 +83,141 @@
...
+    height: 50px;
+    background-color: #fff;
+    color: #396df2;
+    border-radius: 12px;
+    font-size: 17px;
+    font-weight: 500;
+    border: 1px solid #396df2;
+  }
+
+  .image-wrapper {
```
Comment (ridwanediallo) [inline style.css:156]: - [ ] Your snapshots are not correctly aligned, they are moved to the right.
Your section 👇️
Screenshot from 2022-09-24 00-29-46

To help you fix it remove the `  margin-left: -7px;`  from `.image-wrapper` class. And give your snapshot a width of 
`100%` like this 👇️
` .snapshoot {
  width: 100%;
} `

File: index.html [hunk 3]
```diff
@@ -22,18 +26,149 @@ <h3 class="poppins"><a href="#">My Logo</a></h3>
...
+      <li class="about-li">2015</li>
+    </ul>
+    <br/>
+    <p class="poppins-regular">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="poppins-regular" id="languages">
+      <li class="languages">html</li>
+      <li class="languages">css</li>
+      <li class="languages">javascript</li>
+    </ul>
+    <button class="poppins-regular">See Project</button>
```
Comment (ridwanediallo) [inline index.html:86]: - [ ] Please make sure to apply the button styling: Active, Hover and Pressed to all your `buttons`. ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122))

File: index.html [hunk 4]
```diff
@@ -22,18 +26,149 @@ <h3 class="poppins"><a href="#">My Logo</a></h3>
...
+  </section>
+  <br/>
+  <br/>
```
Comment (ridwanediallo) [inline index.html]: [OPTIONAL] : You can remove those `br` tag and give some `margin-bottom` to your `wrappers`.

File: index.html [hunk 5]
```diff
@@ -22,18 +26,149 @@ <h3 class="poppins"><a href="#">My Logo</a></h3>
...
+
+  <section class = "sixth-wrapper">
+
+    <h1 class="poppins">About<br/>Myself</h1>
```
Comment (ridwanediallo) [inline index.html:134]: - [ ] kindly make sure that your `About Myself ` title has the correct colour. 
`color: #172B4D;`

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @Victoria129 , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @Victoria129, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**